tests/utils_test.py

```diff
@@ -106,7 +106,11 @@ def sort_nested(data):
         new_data = {}
         for k, v in data.items():
             if isinstance(v, list):
-                v = sorted(v)
+                try:
+                    v = sorted(v)
+                except TypeError:
+                    # print(f'Warning: Cannot sort {v}')
```

Comment:
nit: commented out code. Printing a warning seems reasonable here, so may I ask why it was commented out?

Reply:
I used this for debugging. This was printing out dictionaries which pop up since calling "sorted()" builtin is undefined behavior when one or more of the entries in the a list is a dict (or, in the case of this code path, OrderedDict). It's not actually a bug; the SPDX format allows this.

I'll remove the dead code.